metaci/release/migrations/0003_populate_releases.py

```diff
@@ -29,49 +31,21 @@ def populate_releases(apps, schema_editor):
     github = login(settings.GITHUB_USERNAME, settings.GITHUB_PASSWORD)
     for info in repos.values():
         repo = github.repository(info['repo'].owner, info['repo'].name) 
-        for release in repo.iter_releases():
-            if release.tag_name in info['tags']:
-                existing = Release.objects.filter(
-                    repo = info['repo'],
-                    git_tag = release.tag_name,
-                )
-                if existing.exists():
-                    continue
-                
-                ref = repo.ref('tags/{}'.format(release.tag_name))
-                rel = Release(
-                    repo = info['repo'],
-                    status = 'published',
-                    version_name = release.name,
-                    version_number = release.name,
-                    git_tag = release.tag_name,
-                    github_release = release.html_url,
-                    release_creation_date = release.created_at,
-                    created_from_commit = ref.object.sha,
-                )
-                sandbox_date = re.findall(
-                    r'^Sandbox orgs: (20[\d][\d]-[\d][\d]-[\d][\d])', release.body
-                )
-                if sandbox_date:
-                    rel.sandbox_push_date = parse_date(sandbox_date[0], '%Y-%m-%d')
-                    
-                prod_date = re.findall(
-                    r'^Production orgs: (20[\d][\d]-[\d][\d]-[\d][\d])', release.body
-                )
-                if prod_date:
-                    rel.production_push_date = parse_date(prod_date[0], '%Y-%m-%d')
-
-                package_version_id = re.findall(r'(04t[\w]{15,18})', release.body)
-                if package_version_id:
-                    rel.package_version_id = package_version_id[0]
-
-                trialforce_id = re.findall(
-                    r'^(0TT[\w]{15,18})', release.body
-                )
-                if trialforce_id:
-                    rel.trialforce_id = trialforce_id[0]
-
-                rel.save()
+        for tag in info['tags']:
```

Comment:
How many releases need to get created in production? Is it still enough to cause problems with the github API?

Reply:
There's 88 releases to be created and the limit is 5000 per hour so we should be good.  There are a bunch of releases in our repositories so the previous implementation which iterated through all releases was the issue.